app/views/users/sessions/new.html.erb

```diff
@@ -65,6 +65,23 @@
   </div>
 </div>
 <script>
+    $(document).ready(()=>{
+      let email_input = $(".user-email-input");
+      let password_input = $(".users-password-input");
+      let login_button = $(".users-login-primary-button");
+
+      toggledButton = () =>{
+        let email_input_value = email_input.val().trim();
+        let password_input_value = password_input.val().trim();
+
+        if((email_input_value !== "") && (password_input_value !== "")){
+          login_button.prop("disabled",false);
+        }
+
+      }
+      email_input.on("input",toggledButton);
+      password_input.on("input",toggledButton);
+    })
```

Comment:
As we are moving to vanilla JS from JQuery, rewrite this part in vanilla JS

Reply:
ok will update that code thank you